changelog/fragments/ansible-helm-multigroup.yaml

```
@@ -0,0 +1,17 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible/Helm, fix multigroup message.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "bugfix"
```

Since no functionality was changed (other than the content of a log message), I don't think I would call this a bugfix.

```suggestion
    kind: "change"
```

---

As I user I would consider it a bug. however, it is fone.